lib/ansible/modules/windows/win_domain_controller.ps1

```diff
@@ -108,11 +106,9 @@ $read_only = Get-AnsibleParam -obj $params -name "read_only" -type "bool" -defau
 $site_name = Get-AnsibleParam -obj $params -name "site_name" -type "str" -failifempty $read_only
 
 $state = Get-AnsibleParam -obj $params -name "state" -validateset ("domain_controller", "member_server") -failifempty $result
-$log_path = Get-AnsibleParam -obj $params -name "log_path"
+$log_path = Get-AnsibleParam -obj $params -name "log_path" -default $null
```

Comment:
The default is `$null`, we typically omit that parameter if we don't have a default to set.

https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/powershell/Ansible.ModuleUtils.Legacy.psm1#L145